src/pudl/transform/eia860.py

```diff
@@ -376,11 +381,7 @@ def _core_eia860__generators_solar(
         .pipe(pudl.helpers.fix_boolean_columns, boolean_columns_to_fix)
         .pipe(pudl.helpers.month_year_to_date)
         .pipe(pudl.helpers.convert_to_date)
-        .pipe(
-            pudl.metadata.classes.Package.from_resource_ids()
-            .get_resource("core_eia860__scd_generators")
-            .encode
-        )
+        .pipe(PUDL_PACKAGE.encode)
```

Comment:
why did you leave in all of the `__generators_*` encoding steps? all of them are both going through harvesting and getting encoded in the _core -> core asset factory. Is it for the `operational_status` column? this can totally be oos for this pr but to me it would make way more sense to assign that column post harvesting.

Reply:
Yes, right now the `operational_status` assignment below depends on having a clean version of the operational status code, and I didn't want to deal with that level of reorganization.